### PR TITLE
Scan the barcode when app starts

### PR DESCRIPTION
**Pull request by Ofir408**
Resovles #7

### CODE REVIEW

File: src/Scheduler.py [hunk 1]
```diff
@@ -41,11 +52,24 @@ def cancel_scheduled_msg(self, scheduled_time_str, contact):
         else:
             print("Failed to cancel msg to contact= {0} at time= {1}".format(contact, scheduled_time_str))
 
+    def init_barcode(self, chrome_path_driver):
```
Comment (ofek286) [inline src/Scheduler.py]: Shouldn't init_barcode be in MsgSender?
The scheduler is only responsible for calling MsgSender when needed.

File: src/Scheduler.py [hunk 2]
```diff
@@ -41,11 +52,24 @@ def cancel_scheduled_msg(self, scheduled_time_str, contact):
...
+        # Be careful with pushing the session to Github!!
+        driver = webdriver.Chrome(executable_path=chrome_path_driver, chrome_options=options)
+
+        driver.get(WHATSAPP_URL)
+        WebDriverWait(driver, 60).until(
+            EC.presence_of_element_located((By.XPATH, "//*[@id=\"app\"]/div/div/div[3]")))
+        driver.quit()
+
 
 if __name__ == '__main__':
```
Comment (ofek286) [inline src/Scheduler.py]: I suggest moving the tests fron the source code into test code in a separate folder.

**Review by ofek286 — CHANGES_REQUESTED** [review]
Please check my comment on Scheduler.py:55

File: src/Scheduler.py (hunk 1 above)
Comment (Ofir408) [reply src/Scheduler.py]: Right

File: src/Scheduler.py (hunk 2 above)
Comment (Ofir408) [reply src/Scheduler.py]: Done

File: MsgSender.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import sys
+import time
+from selenium import webdriver
+
+TIMEOUT = 3
+WHATSAPP_URL = 'https://web.whatsapp.com/'
+
+
+class MsgSender:
```
Comment (ofek286) [inline MsgSender.py]: What is this file?
It looks like it's a copy of src/MessageSender.py

**Review by ofek286 — CHANGES_REQUESTED** [review]

File: MsgSender.py (hunk 3 above)
Comment (Ofir408) [reply MsgSender.py]: Thanks, it was a merge error. 
Fixed

**Review by ofek286 — APPROVED** [review]

**Pull request merged**